packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -75,28 +75,28 @@ mixin RenderProxyBoxMixin<T extends RenderBox> on RenderBox, RenderObjectWithChi
   double computeMinIntrinsicWidth(double height) {
     if (child != null)
       return child!.getMinIntrinsicWidth(height);
-    return 0.0;
+    return computeSizeForNoChild(const BoxConstraints()).width;
```

Comment:
Don't we have to pass in the given `height` to computeSizeForNoChild via the constraints?

Reply:
Good call.  Done.  I also added handling of infinite size returned from `computeSizeForNoChild` (which was the cause of the test failures), returning an intrinsic dimension of 0 in those cases